### PR TITLE
added demo example for Lecture1

### PR DESCRIPTION
**Pull request by ppine7**

### CODE REVIEW

File: fp-java-demo/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
+/bin/
+/gradle/
```
Comment (msmrnv1) [inline fp-java-demo/.gitignore]: should it be a relative path?

Comment (msmrnv1) [reply fp-java-demo/.gitignore]: I also add .DS_Store to ignore in mac

**Comment by esumitra** [comment]
Instead of using a top level folder for the project, lets scope by week or Lecture number to make it easier for students to access resources. Marina, check your email for a proposed folder structure and let me know your feedback.

**Comment by cscie88-ta** [comment]
Looks good to merge. We discussed folder structure into projects and packages for lectures, labs and home works yesterday. The folder structure in this PR is consistent with the folder structure we want to use.

**Comment by dhyaneshm** [comment]
@ppine7 .classpath files, project files, eclipse files should not be commited . can you please remove them ?

**Pull request merged**